### ISSUE TITLE
Reached monthly mobile push message but disconnect from cloud.rocket.chat

### ISSUE THREAD

**Issue by seregasoft** [RocketChat/Rocket.Chat#25154]
### Description:

A message "You have reached your monthly mobile push limit" appears. When you try to close it immediately appears again.
But our server has been disconnected from cloud.rocket.chat and Administration >Push > Enable Gateway disabled.
We successfully receive push notifications to white labelling apps.

### Environment Information:

- Rocket.Chat Server Version: 4.4.0
- Rocket.Chat App Version: WEB 4.4.0

### Steps to reproduce:

1. Login to Rocket.Chat instance by some admin.

### Expected behavior:

No error messages about push limits

### Actual behavior:

"You have reached your monthly mobile push limit"

### Additional context:

Another interesting fact is that this message appears only for two users with the role of Administrator out of seven. And it does not disappear if the user is removed from the Administrator role.

**Comment by seregasoft** [comment]
> Has your workspace been restarted since making that toggle?
Yes, otherwise we would not receive pushes in white labelling applications as I suppose.

> Do you have users that are still using the old mobile app maybe?
Possibly, but I tried to block the user with this problem on the server in to log out from all devices. It did not help, after login the message also appears. Clearing the cache and incognito mode does not solve the problem.

**Comment by geekgonecrazy** [comment]
Might try not only changing to disabled but remove the  value in the gateway address?  We only increment if Rocket.Chat is sending us notifications.  Which it 100% shouldn’t be.
